jaeger_client/tracer.py

```diff
@@ -129,10 +137,16 @@ def start_span(self,
             to avoid extra data copying.
         :param start_time: an explicit Span start time as a unix timestamp per
             time.time()
+        :param ignore_active_span: an explicit flag that ignores the current
+            active :class:`Scope` and creates a root :class:`Span`
 
         :return: Returns an already-started Span instance.
         """
         parent = child_of
+
+        if self.active_span is not None and not ignore_active_span:
```

Comment:
I think it also needs `and not parent`, because if child_of is passed explicitly, the active span should not be inherited (or am I mistaken?)

Reply:
I think you're right, it makes sense. Although one could use `ignore_active_span` together with `child_of` to pass it explicitly. Not exactly intuitive but still an option. The tests don't seem to enforce it in any way (maybe it would be appropriate to add one to do so?).
I added the condition, you might find the formatting controversial (multiline `if`), let me know if you prefer a different way.